.circleci/config.yml

```diff
@@ -140,16 +158,19 @@ commands:
           command: pip install -U pip
       - run:
           name: install cloudify
-          command: pip install -U cloudify==5.0.0 pytest==4.6.3 pyyaml==3.10
+          command: >
+            pip install -U git+https://github.com/cloudify-cosmo/cloudify-common@master#egg=cloudify-common[dispatcher]==master
+            git+https://github.com/cloudify-cosmo/cloudify-cli@master#egg=cloudify-cli==master
+            pytest==4.6.3 pyyaml==3.12
```

Comment:
Let's move python requirements to test-requirements.txt file.

Reply:
done